Web deploy fix

jburel opened the pull request:
Fix command and remove ref to empty file

jburel committed:
Fix command and remove ref to empty file

jburel commented:
@will-moore did you have a chance to look at it?

will-moore commented:
Looks good. @pwalczysko You've been successfully testing these updated commands today, right?

pwalczysko commented:
@will-moore : 

>You've been successfully testing these updated commands today, right?

So for example ``virtualenv omerowebvenv --system-site-packages`` is not included in https://ci.openmicroscopy.org/job/OMERO-DEV-merge-docs/ws/src/omero/_build/html/sysadmins/unix/install-web/walkthrough/omeroweb-install-centos7-ice3.6.html and thus not run today by me at all.

jburel commented:
Looking at the page again, I will be more in favour of removing the commands since we now have the walkthrough links at the top and they do not offer much and add some confusion

jburel committed:
Merge remote-tracking branch 'origin/develop' into web-deploy

jburel committed:
Remove default configuration section

jburel commented:
@will-moore @pwalczysko I have removed the "default" configuration section
I don't think we need to keep it now that we have re-organized the page and added the links to the walkthroughs

mtbc commented on omero/sysadmins/unix/install-web/web-deployment.rst:
```diff
@@ -232,9 +200,7 @@ a minimal configuration:
-`/etc/nginx/conf.d/omero-web.conf`:
-
-.. literalinclude:: nginx-location-manual-wrapper.conf
+`/etc/nginx/conf.d/omero-web.conf`.
```
could be a later PR but maybe more of these should have `:file:` markup

will-moore commented:
Makes sense. Looks good.

jburel committed:
Use :file:

jburel commented:
@mtbc done

jburel commented:
Thanks merging
